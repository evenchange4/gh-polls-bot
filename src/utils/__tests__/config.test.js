// @flow
const config = require('../config');

it('should return config', () => {
  expect(config).toMatchSnapshot();
});
