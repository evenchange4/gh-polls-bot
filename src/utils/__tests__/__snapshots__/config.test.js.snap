// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return config 1`] = `
Object {
  "BASE_URL": "https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod",
  "LABEL": "Polls",
}
`;
